frontend/projects/upgrade/src/assets/js/demo-app.js

```diff
@@ -110,7 +110,7 @@ if (window.self !== window.top) {
       activeWindowEvent.target.removeEventListener(activeWindowEvent.type, activeWindowEvent.callback, true);
```

Comment:
why are demo app changes in this PR?

Reply:
they are mostly related to formatting I guess as I format document with prettier in last so...

Reply:
we should remove linting changes that aren't relevant to the files in factorial feature work

Reply:
however if it's the lint-staged prettier hook that is somehow producing this when you commit your work, let me know, it shouldn't be doing that

Reply:
my prettier isn't working so I don't, I think it has only formatting related changes